.jenkins/pipelines/ci_integrations.groovy

```diff
@@ -0,0 +1,77 @@
+pipeline {
+    agent {
+        docker {
+            image 'testmnrp/dart-converter:2'
+            args '-u root:root'
+        }
+    }
+    options {
+        timeout(time: 2, unit: 'HOURS')
+    }
+    environment {
+        METRICS_PROJECT_ID = "jenkins_ci_integrations"
+    }
+    stages {
+        stage('Git clone') {
+            steps {
+                git branch: 'master', url: 'https://github.com/platform-platform/monorepo.git'
+            }
+        }
+        stage('Get Dependencies') {
+            steps {
+                dir('metrics/ci_integrations'){
+                    sh 'pub get'
+                }
+                script{
+                    isTested = false
+                }
+            }
+        }
+        stage('Parallel Stage') {
+                    parallel {
+                        stage('Run Analyzer') {
+                            steps {
+                                dir('metrics/ci_integrations'){
+                                    sh 'dartanalyzer . --fatal-infos --fatal-warnings'
+                                }
+                            }
+                        }
+                        stage('Run tests') {
+                            steps {
+                                dir('metrics/ci_integrations'){
+                                    sh 'pub run test_coverage --print-test-output --no-badge --port 9498'
+                                }
+                                script{
+                                    isTested = true
+                                }
+                            }
+                        }
+                        stage('Convert coverage report') {
+                            steps {
+                                waitUntil{
+                                    script{
+                                        return isTested
+                                    }
+                                }
+                                dir('metrics/ci_integrations'){
+                                    sh 'coverage_converter lcov -i coverage/lcov.info -o coverage-summary.json'
+                                }
+                            }
+                        post {
+                          always {
+                                archiveArtifacts artifacts: 'metrics/ci_integrations/coverage-summary.json', fingerprint: true
+                            }
+                        }
```

Comment:
Would it be better to move this step out of `parallel`? Please, apply this suggestion to the `coverage_converter.groovy` pipeline as well.